app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -38,6 +38,7 @@ class Api::V2::RepositoriesController < Api::V2::ApiController
       param :unprotected, :bool, :desc => N_("true if this repository can be published via HTTP")
       param :content_type, String, :required => true, :desc => N_("type of repo (either 'yum', 'puppet' or 'docker')")
       param :checksum_type, String, :desc => N_("checksum of the repository, currently 'sha1' & 'sha256' are supported.'")
+      param :docker_upstream_name, String, :desc => N_("name of the upstream docker repository")
```

Comment:
If we refer to this as the `Upstream Repository` in the UI, why not name the API parameter the same for parity?